src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, DrawFigure {
+    protected String color;

```

Comment:
The access modifier is correct - good job.
What needs to be changed is: this value should maintain its type and be converted to string only when necessary: for example when serializing or printing (e.g. when calling draw() method). Otherwise, you lose information about the relation between the value and the enum class.
Please replace the string type with `Color` class on every occurrence.

Reply:
ok, done